### PR TITLE
new architecture refactoring

### PR DESCRIPTION
**Pull request by sk3r**

### CODE REVIEW

File: lib/nagios/probe/tests/occi/test.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
-class Occi::Probe::Tests::TestSuperclass
+class Nagios::Probe::Tests::Occi::Test > Nagios::Probe::TestSuperclass
```
Comment (arax) [inline lib/nagios/probe/tests/occi/test.rb]:
```ruby
class Nagios::Probe::Tests::Occi::Test < Nagios::Probe::TestSuperclass
```

**Comment by arax** [comment]
Why did you close this pull request?

**Comment by sk3r** [comment]
the sole purpose for it's existence was to test whether I can work with branches. This is the old repo, I'll open a request in the shared repository soon.

**Comment by arax** [comment]
Ok :-) Fine with me.

**Pull request closed without merging**